client/my-sites/media-library/filter-bar.jsx

```diff
@@ -63,6 +66,8 @@ export class MediaLibraryFilterBar extends Component {
 		const { translate } = this.props;
 
 		switch ( filter ) {
+			case 'this-post':
+				return translate( 'This Post', { comment: 'Filter label for media list' } );
```

Comment:
RE: `this post` vs. `This Post`, capitalizing visually makes sense with the rest of the filtering options, but perhaps not in the localization context.

We can change to the lower-case string here and apply `text-transform: capitalize` in CSS. What do you think, @folletto?